GameEngine/game_master.py

```diff
@@ -1,10 +1,17 @@
+import sys
```

Comment:
sys and pygame is only needed in `__name__ == "__main__"` how about we move import inside the `if`?